.babelrc

```diff
@@ -1,22 +1,17 @@
 {
-  "presets": ["es2015", "stage-0", "react"],
-  "plugins": ["add-module-exports"],
+  "presets": [],
+  "plugins": ["@babel/plugin-transform-runtime", "transform-class-properties"],
   "env": {
     "production": {
-      "presets": ["react-optimize"],
-      "plugins": [
-        "babel-plugin-transform-remove-console",
-        "babel-plugin-transform-remove-debugger",
-        "babel-plugin-dev-expression"
-      ]
+      "presets": [],
+      "plugins": []
     },
     "development": {
-      "presets": ["react-hmre"]
+      "presets": ["@babel/env"],
+      "plugins": ["transform-class-properties"],
     },
     "test": {
-      "plugins": [
-        ["webpack-loaders", { "config": "webpack.config.node.js", "verbose": false }]
-      ]
+      "presets": ["@babel/env"],
     }
   }
 }
```

Comment:
ParcelJS is smart about knowing which presets/plugins are needed and using those without having to configure `.babelrc`.